### PR TITLE
week 2 assigmnment

### PR DESCRIPTION
**Pull request by dnp87**
Week 2 assignment. Due to me catching up with the course schedule, I decided to leave following problems untended for the time being:
* hardcoded kafka configs
* hardcoded auth service url
* kafka consumption in background thread in mvc/webapi project
* sketchy authorization
* only "parrot created" event is implemented as a proof of concept

### CODE REVIEW

File: aTES/TaskTrackerService/TopicConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Web;
+using TaskTrackerService.Db;
+using LinqToDB;
+
+namespace TaskTrackerService
+{
+    public class TopicConsumer
```
Comment (dnp87) [inline aTES/TaskTrackerService/TopicConsumer.cs:14]: dirty hack...

**Comment by dnp87** [comment]
I failed to convince github to hide some excessive files with another PR: https://github.com/dnp87/ates/pull/6.
Every .config file and every .cs file in /Db folders of AuthService and TaskTrackers service bears no significance to logic and should be treated as boilerplate files.

**Comment by dnp87** [comment]
@davydovanton, @f213, week 2 assignment is ready for review

**Comment by davydovanton** [comment]
Привет! Все посмотрел.

На что обратить внимание на разборе:

- Нейминг топиков/очередей
- что в событии должно быть из данных

**Pull request merged**